x-pack/plugins/observability_solution/observability_ai_assistant_app/public/hooks/use_previous_prompts.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { uniq } from 'lodash';
+import useLocalStorage from 'react-use/lib/useLocalStorage';
+
+const AI_ASSISTANT_LAST_USED_PROMPT_STORAGE = 'ai-assistant-last-used-prompts';
+
+export function usePreviousPrompts() {
+  const [previousPrompts = [], setPreviousPrompts] = useLocalStorage(
+    AI_ASSISTANT_LAST_USED_PROMPT_STORAGE,
+    [] as string[]
+  );
+
+  return {
+    previousPrompts,
+    addPrompt: (prompt: string) => {
```

Comment:
small naming nit, `addPrompt` is not super clear outside of the context of this hook. Perhaps rename this hook to `useLastUsedPrompts` and `addPrompt` to `addLastUsed`?

Reply:
Sure